src/main/java/com/nha/abdm/wrapper/hip/hrp/WorkflowManager.java

```diff
@@ -170,4 +177,28 @@ public FacadeResponse addPatients(List<Patient> patients) {
   public FacadeResponse confirmAuthOtp(VerifyOTP verifyOTP) throws IllegalDataStateException {
     return hipLinkInterface.confirmAuthOtp(verifyOTP);
   }
+
+  public void initiateConsentOnNotify(HIPNotifyRequest hipNotifyRequest)
+      throws IllegalDataStateException {
+    if (hipNotifyRequest != null) {
+      log.info(hipNotifyRequest.toString());
+      consentInterface.notifyOnReceived(hipNotifyRequest);
```

Comment:
again the method name should be hipNotify.. `consentInterface.hipNotify(hipNotifyRequest)`